addons/clones/spriteCount.js

```diff
@@ -0,0 +1,68 @@
+export default async function ({ addon, console }) {
+  const vm = addon.tab.traps.vm;
+  const ogCloneCounter = vm.runtime.changeCloneCounter;
+
+  function updateCounts(remove) {
+    let counts = {};
+
+    vm.runtime.targets
+      .filter((target) => !target.isOriginal)
+      .map((target) => target.sprite.name)
+      .forEach((target) => {
+        if (!counts[target]) counts[target] = 1;
+        else counts[target] += 1;
+      });
+
+    const spriteNames = Array.from(
+      document
+        .querySelector("[class*=sprite-selector_items-wrapper]")
+        .querySelectorAll("[class*=sprite-selector-item_sprite-name]")
+    );
+
+    spriteNames.forEach((spriteName) => {
+      spriteName.querySelector(".clone-count")?.remove();
+      if ((counts[spriteName.innerText.split("\n")[0]] !== undefined) & !remove) {
+        const count = document.createElement("div");
+
+        count.classList.add("clone-count");
+        count.innerText = `(${counts[spriteName.innerText.split("\n")[0]]} clones)`;
+        spriteName.appendChild(count);
+      }
+    });
+  }
+
+  function polute() {
+    if (!addon.settings.get("showSpriteCount")) return;
+
+    vm.runtime.changeCloneCounter = (e) => {
+      ogCloneCounter.call(vm.runtime, e);
+
+      setTimeout(() => {
+        updateCounts();
+      }, 100);
+    };
+
+    updateCounts(false);
+  }
+
+  function unpolute() {
+    vm.runtime.changeCloneCounter = ogCloneCounter;
+    updateCounts(true);
+  }
```

Comment:
dont unpollute :) use the if check.

Reply:
Yea I discovered that literally 2 days ago lmao, thanks for the reminder

Reply:
Doneeee